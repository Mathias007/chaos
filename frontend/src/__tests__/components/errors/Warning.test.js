import React from "react";
import renderer from "react-test-renderer";
import Warning from "../../../pages/errors/Warning";

describe("errorsWarning test", () => {
    it("errorsWarning should match snapshot", () => {
        const component = renderer.create(<Warning />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
