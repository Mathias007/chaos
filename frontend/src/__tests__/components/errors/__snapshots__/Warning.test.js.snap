// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errorsWarning test errorsWarning should match snapshot 1`] = `
<div
  className="ant-result ant-result-warning"
>
  <div
    className="ant-result-icon"
  >
    <i
      aria-label="icon: warning"
      className="anticon anticon-warning"
    >
      <svg
        aria-hidden="true"
        className=""
        data-icon="warning"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M955.7 856l-416-720c-6.2-10.7-16.9-16-27.7-16s-21.6 5.3-27.7 16l-416 720C56 877.4 71.4 904 96 904h832c24.6 0 40-26.6 27.7-48zM480 416c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v184c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V416zm32 352a48.01 48.01 0 0 1 0-96 48.01 48.01 0 0 1 0 96z"
        />
      </svg>
    </i>
  </div>
  <div
    className="ant-result-title"
  >
    There are some problems with your operation.
  </div>
  <div
    className="ant-result-extra"
  >
    <button
      className="ant-btn ant-btn-primary"
      onClick={[Function]}
      type="button"
    >
      <span>
        Go Console
      </span>
    </button>
  </div>
</div>
`;
